study.md

```diff
@@ -27,17 +28,17 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+HTML & View Source.
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+Kinda because the DOM is a representation of your HTML resource.
 ```
```

Comment:
you are totally write by saying kinda. What we see in Dev Tools is actually a visual representation of the DOM.